JavaScript/cluster/public/js/main.js

```diff
@@ -0,0 +1,70 @@
+// this API Key will only work on Algorithmia's website; get your own key at https://algorithmia.com/user#credentials
+var apiKey = 'simRN+PoQevVd6ePYU9t/kEH4MM1'
```

Comment:
:bulb: If possible, let's try to reuse a demo key, otherwise let's add a quick note about who owns this key and its perms.

Reply:
Added to tracking sheet!